src/main.rs

```diff
@@ -0,0 +1,83 @@
+use mdbook::renderer::RenderContext;
+use serde::Deserialize;
+use std::collections::BTreeMap;
+use std::io;
+
+#[derive(Deserialize)]
+struct I18nConfiguration {
+    languages: BTreeMap<String, String>,
+    default_language: Option<String>,
+
+    #[serde(default)]
+    translate_all_languages: bool,
+    #[serde(default)]
+    move_translations_to_html_directory: bool,
+}
+
+fn main() {
+    let mut stdin = io::stdin();
+
+    // Get the configs
+    let ctx = RenderContext::from_json(&mut stdin).unwrap();
+    let i18n_config: I18nConfiguration = ctx
+        .config
+        .get_deserialized_opt("output.i18n-helpers")
```

Comment:
There is a [get_renderer](https://docs.rs/mdbook/latest/mdbook/config/struct.Config.html#method.get_renderer) method available, but I guess it doesn't improve much?

Reply:
It returns a `HashMap`, it's nicer to have a typed `struct`